Source/JavaScriptCore/jit/ExecutableAllocator.h

```diff
@@ -119,6 +119,71 @@ JS_EXPORT_PRIVATE void dumpJITMemory(const void*, const void*, size_t);
 JS_EXPORT_PRIVATE void* performJITMemcpyWithMProtect(void *dst, const void *src, size_t n);
 #endif
 
+#if ENABLE(JIT_SCAN_ASSEMBLER_BUFFER_FOR_ZEROES)
+// This helps with error logging in the case of a crash, as it places the crash
+// PC at the point in the instruction stream which would be corrupted --
+// while preserving the stack so that we can use that to figure out who might be
+// responsible for generating the corrupted instructions.
+template<typename T, typename U, typename V>
+static NEVER_INLINE NO_RETURN_DUE_TO_CRASH NOT_TAIL_CALLED void dieByJumpingIntoJITBufferWithInfo(int line, void* buffer, size_t offset, size_t size, T info1, U info2, V info3)
```

Comment:
Nit: You can just do `auto info1, auto info2, auto info3`

Reply:
True! Will fix